osmaxx-py/osmaxx/utilities/shortcuts.py

```diff
@@ -32,3 +38,44 @@ def get_list_or_none(klass, *args, **kwargs):
     if not object_list:
         return None
     return object_list
+
+
+class Emissary:
+    def __init__(self, user):
```

Comment:
Name the constructor arguments (and the corresponding fields) according to the roles they have w.r.t. the created emissary object rather than their type. (E.g. `recepient` instead of `user`.)

Otherwise construction looks like an Emissary is a special user that wraps a normal user.